### PR TITLE
Update cloud-enabling-continuous-integration-with-github.md

### PR DESCRIPTION
**Pull request by jeremyyeo**
## Description & motivation
dbt Cloud cannot checkout commits from a fork of the repo. See ZD ticket 23941 (internal only) for more info.

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [x] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

### CODE REVIEW

**Comment by mirnawong1** [comment]
hey @jeremyyeo - great write up! i just wanted to double check this was still up to date?

**Comment by jeremyyeo** [comment]
> hey @jeremyyeo - great write up! i just wanted to double check this was still up to date?

yea we should still merge this.

File: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-enabling-continuous-integration-with-github.md [hunk 1]
```diff
@@ -141,3 +141,19 @@ Select the **GitHub**, **GitLab**, or **AzureDevOps** tab and reselect your repo
 If you receive a schema-related error message referencing a *previous* PR, this is usually an indicator that you are not using a production job for your deferral and are instead using *self*.  If the prior PR has already been merged, the prior PR's schema may have been dropped by the time the Slim CI job for the current PR is kicked off.
 
 To fix this issue, select a production job run to defer to instead of self.
+
+### Runs are failing at the "Clone Git Repository" step
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-enabling-continuous-integration-with-github.md]:
```suggestion
### Production job run failing at the **Clone Git Repository** step
```

File: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-enabling-continuous-integration-with-github.md [hunk 2]
```diff
@@ -141,3 +141,19 @@ Select the **GitHub**, **GitLab**, or **AzureDevOps** tab and reselect your repo
...
+
+If you receive an error message like the following during the "Clone Git Repository" step:
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-enabling-continuous-integration-with-github.md]:
```suggestion
dbt Cloud can only checkout commits that belong to the original repository. dbt Cloud _cannot_ checkout commits that belong to a fork of that repository.

If you receive the following error message at the **Clone Git Repository** step of your job run:```

File: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-enabling-continuous-integration-with-github.md [hunk 3]
```diff
@@ -141,3 +141,19 @@ Select the **GitHub**, **GitLab**, or **AzureDevOps** tab and reselect your repo
...
+Error message:
+Cloning into '/tmp/jobs/123456/target'...
+Successfully cloned repository.
+Checking out to e845be54e6dc72342d5a8f814c8b3316ee220312...
+Failed to checkout to specified revision.
+git checkout e845be54e6dc72342d5a8f814c8b3316ee220312
+fatal: reference is not a tree: e845be54e6dc72342d5a8f814c8b3316ee220312
+```
+
+Double check that your PR is not trying to merge using a commit that belongs to a fork of the repository attached to your dbt project. dbt Cloud can only checkout commits that belong to the original repository and not commits that belong to a fork of that repository.
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-enabling-continuous-integration-with-github.md]:
```suggestion
Double-check that your PR isn't trying to merge using a commit that belongs to a fork of the repository attached to your dbt project.
```

**Review by mirnawong1 — APPROVED** [review]

File: website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-enabling-continuous-integration-with-github.md [hunk 4]
```diff
@@ -141,3 +141,21 @@ Select the **GitHub**, **GitLab**, or **AzureDevOps** tab and reselect your repo
 If you receive a schema-related error message referencing a *previous* PR, this is usually an indicator that you are not using a production job for your deferral and are instead using *self*.  If the prior PR has already been merged, the prior PR's schema may have been dropped by the time the Slim CI job for the current PR is kicked off.
 
 To fix this issue, select a production job run to defer to instead of self.
+
+### Production job run failing at the **Clone Git Repository** step
```
Comment (mirnawong1) [inline website/docs/docs/dbt-cloud/using-dbt-cloud/cloud-enabling-continuous-integration-with-github.md]:
```suggestion
### Production job runs failing at the **Clone Git Repository** step
```

**Comment by mirnawong1** [comment]
> > hey @jeremyyeo - great write up! i just wanted to double check this was still up to date?
> 
> yea we should still merge this.

thanks @jeremyyeo and merged w small tiny tweaks! great callout on this!

**Pull request merged**